basic-sql-commands.md

```diff
@@ -98,14 +98,14 @@ To verify that all columns were created correctly, use the [`SHOW COLUMNS FROM`]
 SHOW COLUMNS FROM table1;
 ```
 ```
-+----------+--------+-------+---------------------------+
-|  Field   |  Type  | Null  |          Default          |
-+----------+--------+-------+---------------------------+
-| column_a | INT    | true  | NULL                      |
-| column_b | FLOAT  | true  | NULL                      |
-| column_c | DATE   | true  | NULL                      |
-| column_d | STRING | true  | NULL                      |
-+----------+--------+-------+---------------------------+
++------------+----------+-------+---------+
+|   Field    |   Type   | Null  | Default |
++------------+----------+-------+---------+
+| "column_a" | "INT"    | false | NULL    |
```

Comment:
The types shouldn't have quotes.. Do they show up with quotes in our sql "shell"?